theano/gpuarray/dnn.py

```diff
@@ -2732,6 +2732,176 @@ def local_abstractconv3d_cudnn_graph(op, context_name, inputs, outputs):
     return [rval]
 
 
+class GpuDnnSpatialTfDesc(COp):
+
+    """
+    This Op builds a spatial transformer descriptor for use in spatial transformer network
+    operations.
```

Comment:
Add doc for op attributes (dimensions and dtype), and describe the expected format for dimensions (NCHW),